libr/bin/format/elf/elf.c

```diff
@@ -2118,8 +2192,26 @@ char* Elf_(r_bin_elf_get_head_flag)(ELFOBJ *bin) {
 		default :                   return strdup (" Unknown mips ISA");
 		}
 	}
-	//TODO: Fill other arch 
-	return strdup("unknown_flag");
+	return NULL;
+}
+
+char* Elf_(r_bin_elf_get_head_flag)(ELFOBJ *bin) {
+	char *head_flag = r_str_new("");
+	char *str = NULL;
+	str = Elf_(r_bin_elf_get_cpu)(bin);
+	if (str) {
+		r_str_append(head_flag, str);
+	}
+	str = Elf_(r_bin_elf_get_features)(bin);
+	if (str) {
+		r_str_append(head_flag, " ");
```

Comment:
Same here, missing psaces

Reply:
What about using csv like asm.cpu instaed of ssv?

Reply:
There are many kinds of information in head_flag (ISA, ABI, etc.)

Reply:
This could result in a string that starts with a space. Also im asking here to use a comma a separator instead of a space

Reply:
okay~